src/Refactoring-Tests-Core/RBConditionTest.class.st

```diff
@@ -122,13 +122,26 @@ RBConditionTest >> testHierarchyOfReferencesInstanceVariable [
 
 	| testClass |
 	testClass := rbNamespace classNamed: self class name.
-
+	self halt.
```

Comment:
This halt was left here, not sure if it is on purpose?

Reply:
I will fix it too.